### ISSUE TITLE
Problem with pages containing content builder, no longer editable

### ISSUE THREAD

**Issue by albertorimonda** [chrisblakley/Nebula#1810]
Hello
We have a problem with our WordPress site, Version 5.0.3, theme NEBULA.
We do not know how long the problem exists, if we go to edit a page, and modify a present Content Builder, it loses its Content Builder property.
Example: Edit on a PORTFOLIO content builder, modification of the description, Save Changes, the word PORTFOLIO disappears from the section of the present Content Builder.
If we refresh the page, go out and re-enter, no changes have been applied.
Basically, we can no longer edit pages containing Content Builder.
I hope I was clear.
Has anyone already had a similar problem?
Thank you

**Comment by albertorimonda** [comment]
Hi Chris, thanks for the reply.
We installed the Classic Editor plugin, so we checked the problem with both the Block editor and the Classic. The indicated portfolio is a portfolio with some customizations, in the proposed fields. The same problem occurs with all objects in the Content Builder: Divider, Text Block, Gallery. Thanks

**Comment by chrisblakley** [comment]
Thanks, but I'm still a little unclear– what is the content builder you're referring to?

I will check it with both the Classic Editor and Block Editor, but in order to do so I'll need to install which ever Content Builder you are using.

Also, are you using the latest version of Nebula? It is visible in the bottom right of each admin page. I do see you are on the latest version of WordPress.

**Comment by chrisblakley** [comment]
Nebula does not include anything regarding a "Content Builder" itself, so it sounds like it could be a plugin. Can you take a screenshot so I can see what it looks like?

Are you sure you have Nebula 1.5? That is a very old version of Nebula (roughly 2014). You may need to download the latest version and install that (after backing up your site/DB) and see if that helps.

The error log is pointing to WordPress Core, but that could have been triggered from something that Nebula was using a long time ago that is no longer supported by WordPress core.

**Comment by albertorimonda** [comment]
![nebula](https://user-images.githubusercontent.com/46820826/51482686-6bf50e00-1d97-11e9-936c-83bb1b563965.PNG)
![content_builder1](https://user-images.githubusercontent.com/46820826/51482694-6eeffe80-1d97-11e9-892a-38b965e4f5bd.PNG)
![content_builder2](https://user-images.githubusercontent.com/46820826/51482700-70b9c200-1d97-11e9-8816-bc901140f29c.PNG)
In the image Content_Builder2 you see what happens when you make a change to the existing element: you lose the feature of the element, in this case a Portfolio. Thanks

**Comment by chrisblakley** [comment]
Ah, that appears to be a different theme that is also named Nebula. Here's what I could find on that:
https://themeforest.net/item/nebula-responsive-multipurpose-theme/7112978

You'll need to contact that developer for support on this problem.
